cluster/manifests/kube-static-egress-controller/01-rbac.yaml

```diff
@@ -24,6 +24,18 @@ roleRef:
   kind: ClusterRole
   name: kube-static-egress-controller
 subjects:
-- kind: ServiceAccount
-  name: kube-static-egress-controller
+- apiGroup: rbac.authorization.k8s.io
+  kind: User
+  name: zalando-iam:zalando:service:stups_kubernetes
```

Comment:
Why do we need this subject in the binding?

Reply:
That's basically the username you get when using the PCS. So it gives the container that mounts the PCS the specific RBAC permissions by this binding.

Reply:
Ah, okay. Makes sense 🙂